### ISSUE TITLE
Docker Container for 6.4.10.5 is broken

### ISSUE THREAD

**Issue by wendelb** [CollaboraOnline/online#2978]
**Describe the bug**
The Docker Container for 6.4.10.5 is broken. SSL Certificates cannot be validated (only tried with LetsEncrypt certs).

**To Reproduce**
1. Run the docker container with Nextcloud Integration
2. Open a document
3. See the loading spinner never advancing

**Expected behavior**
The document to be edited

**Actual behavior**

The main part from the log is in line 2: `Failed to verify the certificate of [<MY-Domain>]`

```
wsd-00007-00069 2021-07-27 21:53:24.441367 [ docbroker_004 ] WRN  Waking up dead poll thread [HttpSynReqPoll], started: false, finished: false| ./net/Socket.hpp:682                
wsd-00007-00069 2021-07-27 21:53:24.447258 [ docbroker_004 ] WRN  Failed to verify the certificate of [<MY-Domain>]| ./net/SslSocket.hpp:193                                   
wsd-00007-00069 2021-07-27 21:53:24.447409 [ docbroker_004 ] ERR  Socket #31 SSL BIO error: error:1420C0CF:SSL routines:ssl_write_internal:protocol is shutdown (0: Success)| ./net/SslSocket.hpp:327                                                                                                                                                                    
wsd-00007-00069 2021-07-27 21:53:24.447460 [ docbroker_004 ] ERR  Error while handling poll for socket #31 in HttpSynReqPoll: error:1420C0CF:SSL routines:ssl_write_internal:protocol is shutdown| net/Socket.cpp:424                                                                                                                                                    
wsd-00007-00069 2021-07-27 21:53:24.447518 [ docbroker_004 ] ERR  WOPI::CheckFileInfo failed for URI [https://<MY-DOMAIN>/index.php/apps/richdocuments/wopi/files/123_ABC?access_token=ABC&access_token_ttl=0&permission=edit]: 0 . Headers:      Body: []| wsd/Storage.cpp:703                                               
wsd-00007-00069 2021-07-27 21:53:24.447560 [ docbroker_004 ] ERR  loading document exception: WOPI::CheckFileInfo failed: | wsd/DocumentBroker.cpp:1893                             
wsd-00007-00069 2021-07-27 21:53:24.447589 [ docbroker_004 ] ERR  Failed to add session to [/index.php/apps/richdocuments/wopi/files/123_ABC] with URI [https://<MY-DOMAIN>/index.php/apps/richdocuments/wopi/files/123_ABC?access_token=ABC&access_token_ttl=0&permission=edit]: WOPI::CheckFileInfo failed: | wsd/DocumentBroker.cpp:1855                                                                                                                                                               
wsd-00007-00069 2021-07-27 21:53:24.447607 [ docbroker_004 ] ERR  Storage error while starting session on /index.php/apps/richdocuments/wopi/files/123_ABC for socket #22. T
erminating connection. Error: WOPI::CheckFileInfo failed: | wsd/LOOLWSD.cpp:3592                                                                 
wsd-00007-00069 2021-07-27 21:53:24.447697 [ docbroker_004 ] WRN  Ignoring attempted read from 22|./net/Socket.hpp:1021                      
wsd-00007-00069 2021-07-27 21:53:24.447707 [ docbroker_004 ] ERR  Invalid or unknown session [008] to remove.|wsd/DocumentBroker.cpp:1932
wsd-00007-00038 2021-07-27 21:53:26.448094 [ prisoner_poll ] WRN  Prisoner connection disconnected but without valid socket.| wsd/LOOLWSD.cpp:2231
sh: 1: /usr/bin/loolmount: Operation not permitted
frk-00039-00039 2021-07-27 21:53:27.445272 [ forkit ] ERR  Failed to unmount [/opt/lool/child-roots/YrSlclf5hp4G8yV7/tmp].| common/JailUtil.cpp:68
sh: 1: /usr/bin/loolmount: Operation not permitted
frk-00039-00039 2021-07-27 21:53:27.447696 [ forkit ] ERR  Failed to unmount [/opt/lool/child-roots/YrSlclf5hp4G8yV7/lo].| common/JailUtil.cpp:68
sh: 1: /usr/bin/loolmount: Operation not permitted
frk-00039-00039 2021-07-27 21:53:27.450606 [ forkit ] ERR  Failed to unmount [/opt/lool/child-roots/YrSlclf5hp4G8yV7].| common/JailUtil.cpp:68
```

Possible Workaround: Downgrade to 6.4.10.4

See also:
* https://forum.collaboraonline.com/t/err-wopi-checkfileinfo-failed-for-uri-503-service-unavailable/500
* https://forum.collaboraonline.com/t/partially-working-0-connected/488
* https://forum.collaboraonline.com/t/code-connecting-0-cannot-connect-to-ods-document-stops-working/468

**Comment by bjo81** [comment]
I can confirm this issue with some other instances. Upgraded from 6.4.10.4 to 6.4.10.5 because of #2747 so downgrading is not an option.

**Comment by ustrucx** [comment]
Broken for me as well:

All under Unraid version: 6.9.2 
Nextcloud version: 21.0.0
Collabora version: no idea where to look

`wsd-00007-00050 2021-07-28 17:27:43.916268 [ websrv_poll ] ERR Socket #21 SSL BIO error: error:1420C0CF:SSL routines:ssl_write_internal:protocol is shutdown (0: Success)| ./net/SslSocket.hpp:327
wsd-00007-00050 2021-07-28 17:27:43.916296 [ websrv_poll ] ERR Error while handling poll for socket #21 in websrv_poll: error:1420C0CF:SSL routines:ssl_write_internal:protocol is shutdown| net/Socket.cpp:424`

`frk-00046-00046 2021-07-28 17:27:44.614115 [ forkit ] ERR Failed to unmount [/opt/lool/child-roots/PLDy2uyT8eMm2eS9/tmp].| common/JailUtil.cpp:68
sh: 1: /usr/bin/loolmount: Operation not permitted
frk-00046-00046 2021-07-28 17:27:44.629790 [ forkit ] ERR Failed to unmount [/opt/lool/child-roots/PLDy2uyT8eMm2eS9/lo].| common/JailUtil.cpp:68
sh: 1: /usr/bin/loolmount: Operation not permitted
frk-00046-00046 2021-07-28 17:27:44.644916 [ forkit ] ERR Failed to unmount [/opt/lool/child-roots/PLDy2uyT8eMm2eS9].| common/JailUtil.cpp:68
wsd-00007-00108 2021-07-28 17:27:46.373463 [ docbroker_008 ] WRN Waking up dead poll thread [HttpSynReqPoll], started: false, finished: false| ./net/Socket.hpp:682
wsd-00007-00108 2021-07-28 17:27:46.758393 [ docbroker_008 ] WRN Failed to verify the certificate of [cloud.campos.vip]| ./net/SslSocket.hpp:193
wsd-00007-00108 2021-07-28 17:27:46.758481 [ docbroker_008 ] ERR WOPI::CheckFileInfo failed for URI [https://cloud.campos.vip/index.php/apps/richdocuments/wopi/files/24432_occygkpbitjm?access_token=6hlUrblRRGxaTzk9SS4rVNdXYbxnpCIU&access_token_ttl=0&permission=edit]: 0 . Headers: Body: []| wsd/Storage.cpp:703
wsd-00007-00108 2021-07-28 17:27:46.758526 [ docbroker_008 ] ERR loading document exception: WOPI::CheckFileInfo failed: | wsd/DocumentBroker.cpp:1893
wsd-00007-00108 2021-07-28 17:27:46.758542 [ docbroker_008 ] ERR Failed to add session to [/index.php/apps/richdocuments/wopi/files/24432_occygkpbitjm] with URI [https://cloud.campos.vip/index.php/apps/richdocuments/wopi/files/24432_occygkpbitjm?access_token=6hlUrblRRGxaTzk9SS4rVNdXYbxnpCIU&access_token_ttl=0&permission=edit]: WOPI::CheckFileInfo failed: | wsd/DocumentBroker.cpp:1855
wsd-00007-00108 2021-07-28 17:27:46.758557 [ docbroker_008 ] ERR Storage error while starting session on /index.php/apps/richdocuments/wopi/files/24432_occygkpbitjm for socket #21. Terminating connection. Error: WOPI::CheckFileInfo failed: | wsd/LOOLWSD.cpp:3592`

`frk-00046-00046 2021-07-28 17:27:49.390492 [ forkit ] ERR Failed to unmount [/opt/lool/child-roots/OXaROuaMX62AHBPz/tmp].| common/JailUtil.cpp:68
sh: 1: /usr/bin/loolmount: Operation not permitted
frk-00046-00046 2021-07-28 17:27:49.406135 [ forkit ] ERR Failed to unmount [/opt/lool/child-roots/OXaROuaMX62AHBPz/lo].| common/JailUtil.cpp:68
sh: 1: /usr/bin/loolmount: Operation not permitted
frk-00046-00046 2021-07-28 17:27:49.421899 [ forkit ] ERR Failed to unmount [/opt/lool/child-roots/OXaROuaMX62AHBPz].| common/JailUtil.cpp:68`

If you're on Unraid as well and have no idea how to downgrade to get a functional office again go to Unraid -> Docker -> Collabora -> Edit and change the repository from collabora/code to collabora/code:6.4.9.3 and click Apply.

**Comment by wendelb** [comment]
The issue seems to be fixed with 6.4.10.6!
